var/spack/repos/builtin/packages/py-openslide-python/package.py

```diff
@@ -12,9 +12,12 @@ class PyOpenslidePython(PythonPackage):
     homepage = "https://github.com/openslide/openslide-python"
     url      = "https://github.com/openslide/openslide-python/archive/v1.1.1.tar.gz"
 
+    version('1.1.2', sha256='83e064ab4a29658e7ddf86bf1d3e54d2508cc19ece35d55b55519c826e45d83f')
     version('1.1.1', sha256='33c390fe43e3d7d443fafdd66969392d3e9efd2ecd5d4af73c3dbac374485ed5')
 
     depends_on('openslide@3.4.0:')
     depends_on('python@2.6:2.8,3.3:')
-    depends_on('py-setuptools', type='build')
+    # https://github.com/openslide/openslide-python/pull/76
+    depends_on('py-setuptools@:45', type='build', when="@1.1.1")
+    depends_on('py-setuptools',     type='build', when="@1.1.2")
     depends_on('py-pillow-simd+jpeg+jpeg2000+tiff', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('pil', type=('build', 'run'))
```
We have a virtual package for PIL that lets you choose between `py-pillow` and `py-pillow-simd`. The latter only builds on systems with SIMD, it doesn't work on Apple M1 for example. Unfortunately, virtual packages don't support variants yet.